packages/vscode-extension/src/builders/buildAndroid.ts

```diff
@@ -76,14 +78,30 @@ export async function buildAndroid(
   outputChannel: OutputChannel,
   progressListener: (newProgress: number) => void
 ): Promise<AndroidBuildResult> {
+  const { buildScript, env, android } = getLaunchConfiguration();
+
+  if (buildScript?.android) {
+    const apkPath = await runExternalBuild(
```

Comment:
it'd make sense to also pass `env` from launch configuration to external build scripts, no?